pygments/cmdline.py

```diff
@@ -135,6 +135,49 @@ def _print_list(what):
             print("    %s" % docstring_headline(cls))
 
 
+def _print_list_as_json(requested_items):
+    import json
+    result = {}
+    if 'lexer' in requested_items:
+        info = {}
+        for fullname, names, exts, _ in get_all_lexers():
+            info[fullname] = {
+                'names': names,
+                'extensions': exts
```

Comment:
This is usually called `filenames` since it's a glob matching the whole filename.

Any reason you're not returning the mimetypes (third element)?

Reply:
I compared with the current output and it wasn't printed there, so I omitted it here as well, but I have no feelings about adding it to the JSON output :)

Reply:
Changed to filenames, and added mimetypes